app/controllers/application_controller.rb

```diff
@@ -99,7 +69,18 @@ def available_provider?(provider)
   end
   helper_method :available_provider?
 
+  def provider_set?
+    if logged_in? && current_user.provider_id.nil?
+      redirect_to provider_context_path
+    end
+  end
+
   def store_profile(profile = {})
+    # URS login
+
+    # TODO remove this comment with Launchpad feature toggle ticket
```

Comment:
do you know the ticket number for this upcoming ticket?

Reply:
1386 - I didn't think it was necessary to add the ticket number because it was the one that was immediately next.

Reply:
👍